<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureDialog</class>
 <widget class="QFrame" name="CaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Executable</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="exeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>Program</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <item row="0" column="2">
       <widget class="QLineEdit" name="exePath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="exePathBrowse">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="workDirLabel">
        <property name="text">
         <string>Working Directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="workDirBrowse">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cmdLineLabel">
        <property name="text">
         <string>Command-line Arguments</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="envVarLabel">
        <property name="text">
         <string>Environment Variables</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="envVarEdit">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="exePathLabel">
        <property name="text">
         <string>Executable Path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="workDirPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="cmdline">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="envVar">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="injectGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>260</height>
      </size>
     </property>
     <property name="title">
      <string>Process</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="processRefesh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="processWarning">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>106</red>
              <green>104</green>
              <blue>100</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NOTE: Injecting only works when the process has not used the target API</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="processFilter">
        <property name="placeholderText">
         <string>Filter process list by PID or name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTreeView" name="processList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="vulkanLayerWarn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
background-color: #ffffdc;
border: 1px solid black;
}
QToolButton:hover {
background-color: #ddddbe;
}</string>
     </property>
     <property name="text">
      <string>Warning: Vulkan capture is not configured.
Click here to set up Vulkan capture.</string>
     </property>
     <property name="icon">
      <iconset resource="../../Resources/resources.qrc">
       <normaloff>:/information.png</normaloff>:/information.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Capture Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="AllowFullscreen">
        <property name="text">
         <string>Allow Fullscreen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AllowVSync">
        <property name="text">
         <string>Allow VSync</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="DebuggerDelayFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDoubleSpinBox" name="DelayForDebugger">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string comment="seconds"> secs</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DebuggerDelayLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Debugger Delay</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CaptureCallstacks">
        <property name="text">
         <string>Collect Callstacks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CaptureCallstacksOnlyDraws">
        <property name="text">
         <string>Only Drawcall stacks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="APIValidation">
        <property name="text">
         <string>Enable API Validation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HookIntoChildren">
        <property name="text">
         <string>Hook into Children</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveAllInitials">
        <property name="text">
         <string>Save all Initials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RefAllResources">
        <property name="text">
         <string>Ref all Resources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CaptureAllCmdLists">
        <property name="text">
         <string>Capture all Cmd Lists</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="VerifyMapWrites">
        <property name="text">
         <string>Verify Map() Writes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AutoStart">
        <property name="text">
         <string>Auto Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="actionGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="queueFrameCap">
        <property name="text">
         <string>Queue Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="queuedFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="prefix">
         <string comment="Frame number" extracomment="To Capture">Frame </string>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="ActionsSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>596</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="globalGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Global Process Hook</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="globalLabel">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>106</red>
              <green>104</green>
              <blue>100</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Text Set by Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toggleGlobal">
        <property name="text">
         <string>Enable Global Hook</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="globalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>587</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonsFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveSettings">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="buttonsSpace">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>556</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="launch">
        <property name="text">
         <string>Launch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="topVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
